### PR TITLE
Template content

### PR DESCRIPTION
**Pull request by akosgarai**
- Replace the original text content with the requested one.
- Add new block for list items.
- Implement mosaicoConfig hook to be able to update the tinymce config and force the root block to be div.
- Replace thumbnail images.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ![CI](https://github.com/reflexive-communications/remote/workflows/CI/badge.svg)
 
-This extension installs a new base template to the mosaico email template editor civicrm extension. The template is based on the `versafix-1` template, but the `p` html tags have been replaced with `divs`. It was necessary to support the align of the texts. As we experienced, the alignment changes were not applied on the `p` tags.
+This extension installs a new base template to the mosaico email template editor civicrm extension. The template is based on the `versafix-1` template, but the `p` html tags have been replaced with `divs`. It was necessary to support the align of the texts. As we experienced, the alignment changes were not applied on the `p` tags. It also provides a new block in the editor for lists.
```
Comment (semseysandor) [inline README.md]: `divs` could mean `<divs></divs>`. Maybe "`div` tags" instead of "`divs`"?

File: assets/remote-template.html [hunk 2]
```diff
@@ -551,15 +566,15 @@
...
+                          Cím
                           </span>
                         </td>
                       </tr>
                       <tr>
                         <td align="left" style="text-align: left; font-size: 13px; font-family: Arial, Helvetica, sans-serif; color: #3f3f3f;
                           -ko-font-size: @[longTextStyle.size]px; -ko-font-family: @longTextStyle.face; -ko-color: @longTextStyle.color"
                           data-ko-editable="longText" class="long-text links-color">
-                          <div>Far far away, behind the word mountains, far from the countries <a href="">Vokalia and Consonantia</a>, there live the blind texts. Separated they live in Bookmarksgrove right at the coast of the Semantics, a large language ocean. A small river named Duden flows by their place and supplies it with the necessary regelialia.</div>
+                            <div>Ide bármi leírható, ami érdekes lehet. Akár többször is. Viszont, ha felsorolást szeretnél használni, válaszd ki a blokkok közül a <strong>felsorolás</strong> blokkot.<br />Kellemes szerkesztést!</div>
```
Comment (semseysandor) [inline assets/remote-template.html:577]: Are we sure about hungarian?

Comment (akosgarai) [reply assets/remote-template.html:577]: Yes. It was the request.

Comment (semseysandor) [reply assets/remote-template.html:577]: In that case OK.
It just feels strange...
Maybe later when we'll be experts at localization we can change this to english and add the hungarian translation ;)

**Review by semseysandor — APPROVED** [review]

File: assets/remote-template.html (hunk 2 above)
Comment (akosgarai) [reply assets/remote-template.html:577]: I think we can't use localization in the mosaico templates.

**Pull request merged**